docs/routers/fritzbox.md

```diff
@@ -85,3 +84,90 @@ The following settings must be made:
+
+Most OS assign a stable interface identifiers for management purposes to an interface.
+There exist multiple standards and implementations how it is generated and how stable it actually is.
+If you are unsure, consult the documentation of your OS about the configuration.
+The following section will help you picking a suitable IPv6 address of your Pi-hole.
+To show all IPv6 addresses currently in use by your main network interface, open a terminal, substitute `eth0` with your main interface name (or omit) and execute the command
+
+```bash
+ip -6 address show eth0
+```
+
+The output should look like this but with different addresses
+
+```bash
+$ ip -6 address show eth0
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    inet6 fd6f:95dc:3a80:e9b1:503e:b05a:21dc:cf0c/64 scope global temporary dynamic
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 fd6f:95dc:3a80:e9b1:4bc3:7bff:fe67:c175/64 scope global dynamic mngtmpaddr noprefixroute
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 2001:db8::1c5e:22fe:490c:1c31/64 scope global temporary dynamic
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 2001:db8::4bc3:7bff:fe67:c175/64 scope global dynamic mngtmpaddr noprefixroute
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 fe80::4bc3:7bff:fe67:c175/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+When picking an IPv6 address from that list:
+
+* avoid global unicast addresses (GUA) (range 2000::/3)  
+  Your ISP controls your GUA IPv6 prefix it, so it may change, either regularly or on router restarts.
+  In this example, don't use the third and fourth address, starting with 2001:
+* avoid Privacy Extension addresses (marked with temporary)  
+  The interface identifier portion of an IPv6 address is designed to change regularly, on some systems as often as every hour.
+  In this example, avoid the first and the third address.
+* prefer unique local addresses (ULA) (range `fd00::/8`) over the link-local address (range `fe80::/10`)  
+  You can control the ULA prefix and it is static. The later is only valid on a link and cannot be routed.
+  This is fine for very simple setups but will break once packets are routed (like Docker, some WiFi access points, L3 switches, ...).
+
+In this example, these two addresses are usable; `fd6f:95dc:3a80:e9b1:4bc3:7bff:fe67:c175` and `fe80::4bc3:7bff:fe67:c175` (with care).
+
+If your FritzBox doesn't issue an IPv6 ULA prefix yet, refer to the following step which helps you configure an ULA prefix.
+
+### (Optional) Enable ULA addresses
+
+Unique local addresses (ULA) are local IPv6 addresses which are not routed on the internet. They are comparable to the IPv4 private network ranges.
+Not all ISPs assign a static IPv6 prefix. The assigned global prefix usually does not change very often and thus seems to be static.
```

Comment:
I'd skip that sentence completely:
"<strike>Not all ISPs assign a static IPv6 prefix. The assigned global prefix usually does not change very often and thus seems to be static.</strike>"